q2_longitudinal/_utilities.py

```diff
@@ -344,15 +344,20 @@ def _regplot_subplots_from_dataframe(state_column, metric, metadata,
                                      group_by, lowess=False, ci=95,
                                      palette='Set1'):
     '''plot a single regplot for each group in group_by.'''
-    f, axes = plt.subplots(len(group_by), figsize=(6, 18))
+    height = 6 * len(group_by)
```

Comment:
Minor stylistic note, we are now computing `len(group_by)` four times in the span of four lines - this would be a good candidate for splitting out into a new variable:

```python
group_by_count = len(group_by)
height = 6 * group_by_count
....
`